source/deployment-options/offline-installation.rst

```diff
@@ -59,24 +59,27 @@ Download the packages and configuration files
 Install Wazuh components from local files
 -----------------------------------------
 
-In the working directory where you placed ``wazuh-offline.tar.gz`` and ``./wazuh-certificates/``, execute the following command to decompress the installation files:
+#. In the working directory where you placed ``wazuh-offline.tar.gz`` and ``./wazuh-certificates/``, execute the following command to decompress the installation files:
 
-.. code-block:: console
+   .. code-block:: console
+
+      # tar xf wazuh-offline.tar.gz
 
-    # tar xf wazuh-offline.tar.gz
+   Check the SHA512 of the decompressed package files in ``wazuh-offline/wazuh-packages/``. You can find the SHA512 checksums in :doc:`/installation-guide/packages-list`.
 
 Installing the Wazuh indexer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#.  Run the following command to install the Wazuh indexer.
+#.  Run the following commands to install the Wazuh indexer.
 
     .. tabs::
 
         .. group-tab:: RPM
 
             .. code-block:: console
         
-                # rpm -ivh ./wazuh-offline/wazuh-packages/wazuh-indexer*.rpm
+               # rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH
```

Comment:
As the installation is offline, we can not use `rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH` to import the key. Instead, we should use `rpm --import ./wazuh-offline/wazuh-files/GPG-KEY-WAZUH`.